CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ We appreciate your help!
 
 ## Getting started with contribution
 
-* Get started with the Ballerina Integrator by following the [Quick Start Guide](https://ei.docs.wso2.com/en/latest/ballerina-integrator/getting-started/quick-start-guide/). Start trying out our guides and tutorials available in the [Learn](https://ei.docs.wso2.com/en/latest/ballerina-integrator/learn/about-learn/) section.
+* Get started with the Ballerina Integrator by following the [Quick Start Guide](https://ei.docs.wso2.com/en/latest/ballerina-integrator/get-started/quick-start-guide/). Start trying out our guides and tutorials available in the [Learn](https://ei.docs.wso2.com/en/latest/ballerina-integrator/learn/about-learn/) section.
```

Comment:
Let's use this for the Learn section: https://ei.docs.wso2.com/en/latest/ballerina-integrator/learn/use-cases/

Reply:
+1

Reply:
Corrected.